sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -18,6 +18,10 @@ internal class BitmapWidgetRenderer(
 
   private var bitmapWidth = bitmap?.width ?: 0
   private var bitmapHeight = bitmap?.height ?: 0
+  private var halfBitmapWidth = bitmapWidth / 2f
+  private var halfBitmapHeight = bitmapHeight / 2f
```

Comment:
Is bitmapWidth / bitmapHeight needed then? Seems half* are used everywhere now

Reply:
Indeed that is true, I can remove the original values